sinks/gcm/driver.go

```diff
@@ -191,6 +161,7 @@ type timeseriesDescriptor struct {
 type point struct {
 	Start       time.Time `json:"start,omitempty"`
 	End         time.Time `json:"end,omitempty"`
+	Int64Value  int64     `json:"int64Value"`
```

Comment:
Sadly this won't work for now. Since it is not omitempty we'd generate both types which will confuse the GCM API. We can't make them omitempty since 0 is a valid value. We need to have two structs we define for this (probably a base embedded into an int or double type). Easiest thing may be to just use double for all the metrics for now. It should not reduce the accuracy of our numbers I believe.